File: 1832. Check if the Sentence Is Pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        chars=[0]*26
        counter1=0
        counter2=0
        for i in range(0,len(sentence)):
            for j in range(0,26):
                if(sentence[i]==chars[j]):
                    counter1=1
                    #break
            if(counter1==0):
                chars[counter2]=sentence[i]
                counter2=counter2+1
            counter1=0
        
        if(counter2==26):
            return True
        else:
            return False

File: 1832. Check if the Sentence Is Pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        char[] chars=new char[26];
        int len=sentence.length();
        //System.out.println(len);
        //return true;
        int counter1=0;
        int counter2=0;
        for(int i=0;i<len;i++)
        {
            for(int j=0;j<26;j++)
            {
                if(sentence.charAt(i)==chars[j])
                    counter1=1;
            }
            if(counter1==0)
            {
                chars[counter2]=sentence.charAt(i);
                counter2++;
            }
            counter1=0;
        }
        if(counter2==26)
            return true;
        else
            return false;
    }
}
